PeanutPlug.groovy

```diff
@@ -163,8 +96,20 @@ i/*		    input (
                 submitOnChange: true,
                 required: true,
                 range: "1..7200",
-                defaultValue: 60
+                defaultValue: 5
             )
+            
+            input (
+                name: "ReportableCurrentChange",
+                type: "number",
+                title: "Minimum Current Change Report Value",
+                description: "Report Power change greater than XXX amps. (0 - 1000)",

```

Comment:
I think you mean 
Report Current change
instead of 
Report Power change

Reply:
Yep, looks like the same error is in the Voltage setting section, too.

FYI, I am unsure of the units of that minimum change value.  The comments were left mostly intact, but if you know better please correct it.  I just set them all to '1' to mean "small."

Reply:
Sounds good to me.  I do notice that the 'RMS Voltage' seems to log when ever it wants reguardless of the limits that are set so I assume I am missing something.  Would love to hear any thoughts you may have on that.

Reply:
Almost forgot.  I am assuming that a value of 0 for Power/Current/Voltage Change Report Value means 'Don't read these values'??

Reply:
Fixed Current->Power in change #2